.circleci/src/jobs/push-arm-image.yml

```
@@ -0,0 +1,23 @@
+parameters:
+  service:
+    description: 'Service to push'
+    type: string
+resource_class: arm.medium
+machine:
+  image: ubuntu-2204:current
```

```suggestion
machine: true
resource_class: audiusproject/gcp-t2a-standard-4
```
With this change you'll be good to go

---

TYTYTYTYTYTY